### PR TITLE
Implement scheme && add scheme for blackarch

### PR DESCRIPTION
**Pull request by edward-p**

### CODE REVIEW

**Comment by simonesestito** [comment]
Hi! Nice pull request.
Could I know what are the differences between Default scheme and Blackarch scheme?

**Comment by edward-p** [comment]
The difference is mainly about the color (I customized `hoverColor` `focusColor` etc. in `Main.qml`)
Here is my screenshot:

File: install.sh [hunk 1]
```diff
@@ -32,9 +58,18 @@ case $image in
 	2) sudo cp options/logo/neon.png $theme_path/resources/logo.png ;;
 	3) sudo cp options/logo/plasma.png $theme_path/resources/logo.png ;;
 	4) sudo cp options/logo/fedora.png $theme_path/resources/logo.png ;;
+    5) sudo cp options/logo/blackarch.png $theme_path/resources/logo.png ;;
 	*) echo "Unexpected error!"; exit 1 ;;
 esac
 
+# Install scheme
```
Comment (edward-p) [inline install.sh:65]: Each scheme is a customized `Main.qml` and will be installed as `Main.qml`

File: install.sh [hunk 2]
```diff
@@ -4,23 +4,49 @@ theme_path=/usr/share/sddm/themes/dark-sddm-theme
 
 echo "Welcome in theme installer"
 
+logos=(
```
Comment (edward-p) [inline install.sh:7]: Using arrays to store options

File: install.sh [hunk 3]
```diff
@@ -4,23 +4,49 @@ theme_path=/usr/share/sddm/themes/dark-sddm-theme
...
+i=1
+for logo in "${logos[@]}"; do
+    echo "($i) $logo"
+    let "i=i+1"
+done
 
 echo -n "==> "
 read image
-if (( $image < 1 || $image > $max ))
+if (( $image < 1 || $image > ${#logos[@]} ))
```
Comment (edward-p) [inline install.sh:29]: using ${#logo[@]} to get the length

**Comment by edward-p** [comment]
`$ diff default.qml blackarch.qml`
``` diff
--- default.qml
+++ blackarch.qml
@@ -144,6 +144,8 @@
                     radius: 20
                     color: "#77000000"
                     borderColor: "#77000000"
+                    focusColor: "#77c40000"
+                    hoverColor: "#77c40000"
                     textColor: "white"

                     KeyNavigation.tab: password
@@ -156,6 +158,8 @@
                     radius: 20
                     color: "#77000000"
                     borderColor: "#77000000"
+                    focusColor: "#77c40000"
+                    hoverColor: "#77c40000"
                     textColor: "white"

                     focus: true
@@ -209,7 +213,7 @@
                     borderWidth: 0
                     textColor: "white"
                     dropdownColor: "#22000000"
-                    hoverColor: "#77000000"
+                    hoverColor: "#77c40000"
                     width: 500
                     anchors.verticalCenter: parent.verticalCenter
```

**Comment by simonesestito** [comment]
It looks definitely cool. Since I can't test it myself before pushing, I'll ask someone trusted to test this edit (which will work for sure). Then, I'll be happy to merge this pull

**Pull request merged**